scipy/stats/_discrete_distns.py

```diff
@@ -288,6 +288,14 @@ def _cdf(self, x, n, p):
         k = floor(x)
         return special.betainc(n, k+1, p)
 
+    def _logcdf(self, x, n, p):
+        k = floor(x)
+        cdf = self._cdf(k, n, p)
+        cond = cdf > 0.5
+        f1 = lambda k, n, p: np.log1p(-special.betainc(k + 1, n, 1 - p))
+        f2 = lambda k, n, p: np.log(cdf)
```

Comment:
```suggestion
        def f1(k, n, p):
            return np.log1p(-special.betainc(k + 1, n, 1 - p))
        def f2(k, n, p):
            return np.log(cdf)
```
Yes, I know PEP8 recommends `def` instead of `lambda`. But let's go for it and make the code look more complicated to follow PEP8!